webapp/src/components/common/Matrix/components/MatrixGridSynthesis/index.tsx

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (c) 2024, RTE (https://www.rte-france.com)
+ *
+ * See AUTHORS.txt
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This file is part of the Antares project.
+ */
+
+import DataEditor, {
+  GridCellKind,
+  GridColumn,
+  Item,
+  NumberCell,
+  TextCell,
+} from "@glideapps/glide-data-grid";
+import { useMemo } from "react";
+import { darkTheme, readOnlyDarkTheme } from "../../styles";
+import { formatGridNumber } from "../../shared/utils";
+
+type CellValue = number | string;
+
+interface MatrixGridSynthesisProps {
+  data: CellValue[][];
+  columns: GridColumn[];
+  width?: string;
```

Comment:
You can allow number:

```typescript
width?: DataEditorProps["width"];
height?: DataEditorProps["height"];
```

Reply:
true but we usually don't modify this, I will add it tho